cg/constants/compression.py

```diff
@@ -129,6 +129,10 @@
     "nearbyjoey",
 ]
 
+TAXPROFILER_VALIDATION_CASES = [
```

Comment:
```suggestion
TAXPROFILER_VALIDATION_CASES: List[str] = [
```

Reply:
Thanks for the suggestion @henrikstranneheim. I have added `from typing import List` since the tests were failing